python/paddle/fluid/tests/unittests/mkldnn/test_activation_mkldnn_op.py

```diff
@@ -58,6 +59,29 @@ def test_check_grad(self):
             ['X'], 'Out', max_relative_error=0.007, check_dygraph=False)
 
 
+class TestMKLDNNGeluDim2(TestGelu):
+    def setUp(self):
+        super(TestMKLDNNGeluDim2, self).setUp()
+
+        x = np.random.uniform(-1, 1, [11, 17]).astype(np.float32)
+        out = 0.5 * x * (1.0 + np.tanh(
```

Comment:
What about defining the formula as a function and reusing it below?

Reply:
Great idea, but it would be even better to add it in general `test_activation_op.py` and include in UTs when `approximate=True` will be implemented as a part of an API.